README.md

```diff
@@ -123,55 +55,31 @@ Collect the data through teleoperation.
   ```
 Refer to [official hello robot keybindings](https://docs.hello-robot.com/0.2/stretch-tutorials/getting_started/images/xbox.png) to learn how to operate Stretch with the controller. We also provide a `Dataloaders` to load data into torch (WIP). 
 
-### Launching Grasping Demo (outdated)
-
-You need to create a catkin workspace on your workstation in order to run this demo, as this is where we will run [Contact Graspnet](https://github.com/cpaxton/contact_graspnet/tree/cpaxton/devel).
-
-Contact graspnet is downloaded under `third_party/`, but there is a `CATKIN_IGNORE` file in this directory. You want to symlink this file out into your workspace:
-```
-ROSWS=/path/to/ros_ws
-ln -s `rospack find home_robot`/third_party/contact_graspnet $ROSWS/src/contact_graspnet
-```
-... but it actually shouldn't be necessary. What is necessary is to build the grasp service defined in `home_robot` by placing it into `$ROSWS`.
-
+### Syncing code between Robot and Workstation
```

Comment:
the "collecting data" section is not super important and should not be in the root README, imo.